tests/admin_scripts/tests.py

```diff
@@ -2405,6 +2406,25 @@ def test_wrong_args(self):
         self.assertOutput(err, "usage:")
         self.assertOutput(err, "You must provide a project name.")
 
+    def test_output_directory_structure(self):
+
+        dirs = {
+            "testproject": self.test_dir,
+            "testproject1": os.path.join(self.test_dir, "otherdirectory"),
+        }
+        for name_project, dir in dirs.items():
+            with self.subTest(name_project=name_project, dir=dir):
+                args = ["startproject", name_project]
+                if not os.path.exists(dir):
+                    os.mkdir(dir)
+                    args.append(dir)
+                out, err = self.run_django_admin(args)
+
+                # We only check for a subset of the directory structure
```

Comment:
I think it's better to assert all of it? 🤔

Reply:
I was thinking, since the lower level tests verified most of the function's behaviour, another complete check would be too much?
Then again it does make more sense seeing the full directory structure when testing.